### PR TITLE
Make titles and spacings consistent across tool options

### LINKED ISSUE #1368: Checkboxes are not consistent across Tool Options
**Synfig version**:
master 7d0ef7e

**Issue description**:
1. Create a new animation
2. Select the `Scale Tool` /  `Rotate Tool` / `Width Tool` / `Sketch Tool` in the `Toolbox Panel`
3. Look at the `Tool Options` Panel, e.g. `Sketch Tool`:
Sketch_Tool

The checkbox is positioned on the left, while in all the other Tools the checkboxes are placed on the right:
Spline_Tool

I suggest to place all the checkboxes in a consistent way across all the Tool Options to the right.

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/91175539-checkboxes-are-not-consistent-across-tool-options?utm_campaign=plugin&utm_content=tracker%2F321991&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F321991&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by FirasH**
Fixes #1368

### CODE REVIEW

File: synfig-studio/src/gui/states/state_lasso.cpp [hunk 1]
```diff
@@ -615,7 +615,7 @@ StateLasso_Context::StateLasso_Context(CanvasView* canvas_view):
 	/* Set up the tool options dialog */
 
 	// 0, title
-	title_label.set_label(_("Lasso"));
+	title_label.set_label(_("Cutout Tool"));
```
Comment (rodolforg) [inline synfig-studio/src/gui/states/state_lasso.cpp:618]: Great! :)

File: synfig-studio/src/gui/states/state_mirror.cpp [hunk 2]
```diff
@@ -71,6 +71,8 @@ enum Axis {
 	AXIS_Y
 } ;
 
+#define GAP (3)
```
Comment (rodolforg) [inline synfig-studio/src/gui/states/state_mirror.cpp:74]: No need to use `#define`

Just put a `const int gap = 3;` inside the method :)

Comment (FirasH) [reply synfig-studio/src/gui/states/state_mirror.cpp:74]: `#define GAP (3)` is used for all the Tools (repeated for each one in their `.cpp` file).
Do you suggest to substitute all those instances?

**Comment by rodolforg** [comment]
Can anybody restart Travis CI here? :P 18 days queued

**Comment by ice0** [comment]
Done :)

**Comment by rodolforg** [comment]
It stalled again :(

**Comment by rodolforg** [comment]
Why two "Travis CI - Pull Request" anyway?

**Comment by ice0** [comment]
It compiles fine. Just wait until @morevnaproject check it.

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**